app/javascript/packs/ResetPasswordPage.js

```diff
@@ -0,0 +1,52 @@
+import '@babel/polyfill'
```

Comment:
@duduribeiro By convention, we don't render React components directly from the `javascript/packs` directory. In `packs` we only get from rails the props needed and pass them down.
The React components are located in the `javascript/src` directory and  we use a helper called `createReactWrapper`.
I'll create a PR pointing to this one to fix the issues.